sdks/python/tox.ini

```diff
@@ -50,7 +50,7 @@ commands =
   pip --version
   {toxinidir}/run_tox_cleanup.sh
   python apache_beam/examples/complete/autocomplete_test.py
-  python setup.py test
+  python setup.py nosetests --with-xunit
```

Comment:
Will each run's `nosetests.xml` output file overwrite the others'?

Reply:
I believe it will since they will overwrite each other. You would only get the test results from the last run.